Mathlib/Data/Nat/Digits.lean

```diff
@@ -633,6 +636,22 @@ theorem ofDigits_mod (b k : ℕ) (L : List ℕ) : ofDigits b L % k = ofDigits (b
   ofDigits_modEq b k L
 #align nat.of_digits_mod Nat.ofDigits_mod
 
+theorem ofDigits_mod_eq_head! (b : ℕ) (l : List ℕ) : ofDigits b l % b = l.head! %b := by
```

Comment:
Do we have `Nat.ModEq` available in this file?

Reply:
If we do not, then we should fix the import tree.

Reply:
yes, should we use it somehow?